docs/Training-PPO.md

```diff
@@ -70,8 +70,8 @@ Typical Range: `0.9` - `0.95`
 
 ### Buffer Size
 
-`buffer_size` corresponds to how many experiences (agent observations, actions
-and rewards obtained) should be collected before we do any learning or updating
+`buffer_size` corresponds to how many experiences should be collected before
```

Comment:
Can you move this change to my release branch: https://github.com/Unity-Technologies/ml-agents/tree/release-mm-github-docs